documentation/modules/rn-2.5.adoc

```diff
@@ -124,40 +128,113 @@ When migrating VMs that are installed with RHEL 9 as guest operating system from
+
+
+.CVE-2023-39321: Processing an incomplete post-handshake message for a QUIC connection can cause a panic
+
+A flaw was found in Golang. This flaw means {project-short} versions before {project-short} 2.5.3 are vulnerable to processing an incomplete post-handshake message for a QUIC connection, which causes a panic. link:https://issues.redhat.com/browse/MTV-693[(MTV-693)]
+
+This issue has been resolved in {project-short} 2.5.3. It is advised to update to this version of {project-short} or later.
+
+For more information, see link:https://access.redhat.com/security/cve/CVE-2023-39321[CVE-2023-39321].
+
+
+.CVE-2023-39319: Flaw in html/template package
+
+A flaw was found in the Golang `html/template` package used in {project-short}. This flaw means {project-short} versions before {project-short} 2.5.3 are vulnerable, as the `html/template` package did not properly handle occurrences of `<script`, `<!--`, and `</script` within JavaScript literals in `<script>` contexts. This flaw could cause the template parser to improperly consider script contexts to be terminated early, causing actions to be improperly escaped, which could be leveraged to perform an `XSS` attack. link:https://issues.redhat.com/browse/MTV-693[(MTV-693)]
+
+This issue has been resolved in {project-short} 2.5.3. It is advised to update to this version of {project-short} or later.
+
+For more information, see link:https://access.redhat.com/security/cve/CVE-2023-39319[CVE-2023-39319].
+
+
+.CVE-2023-39318: Flaw in html/template package
+
+A flaw was found in the Golang `html/template` package used in {project-short}. This flaw means {project-short} versions before {project-short} 2.5.3 are vulnerable as the `html/template` package did not properly handle HMTL-like `""` comment tokens, nor hashbang `\#!` comment tokens. This flaw could cause the template parser to improperly interpret the contents of `<script>` contexts, causing actions to be improperly escaped, which could be leveraged to perform an `XSS` attack. link:https://issues.redhat.com/browse/MTV-693[(MTV-693)]
+
+This issue has been resolved in {project-short} 2.5.3. It is advised to update to this version of {project-short} or later.
+
+For more information, see link:https://access.redhat.com/security/cve/CVE-2023-39318[CVE-2023-39318].
+
+
+.Logs archive file downloaded from UI includes logs related to deleted migration plan/VM
+
+In previous releases of {project-short} {project-version}, the logs file downloaded from UI could contain logs that are related to a previous migration plan. link:https://issues.redhat.com/browse/MTV-783[(MTV-783)]
+
+This issue has been resolved in {project-short} 2.5.3.
+
+
+.Extending a VM disk in RHV is not reflected in the MTV inventory
+
+In previous releases of {project-short} {project-version},
+
+link:https://issues.redhat.com/browse/MTV-830[(MTV-830)]
+
+This issue has been resolved in {project-short} 2.5.3.
```

Comment:
@ahadas & @qiyuann - please could you advise what else you want me to cover for this as I do not really understand it? Thanks

Reply:
In previous releases of {project-short} {project-version}, the size of disks that are extended in RHV has not been properly monitored. This resulted in failure to migrate VMs with extended disks from a RHV provider.